tasks/src/reinforcement-learning/about.md

```diff
@@ -0,0 +1,45 @@
+## Use Cases
+
+You can contribute this area with common use cases of the task! 
+
+## Task Variants 
+
+This place can be filled with variants of this task if there's any. 
+
+## Glossary
+
+<!-- ![RL Loop](https://huggingface.co/blog/assets/63_deep_rl_intro/RL_process.jpg "Agent Environment Interaction") TODO: Uncomment image for visual understanding if it fits within the page--> 
+
+
+**Agent:** The learner and the decision maker
+
+**Environment:** The part of the world the agent interacts, comprising everything outside the agent
+
+**State:** Information the agent receives from the environment. In the case of a video game it can be a frame (a screenshot), in the case of a chess playing agent it can be the board position, in the case of a trading agent it can be the price of a certain stock, etc

```

Comment:
```suggestion
**State:** Information the agent receives from the environment. In the case of a video game it can be a frame (a screenshot), in the case of a chess playing agent it can be the board position, in the case of a trading agent it can be the price of a certain stock.

```